app/views/menus/_card.html.erb

```diff
@@ -1,5 +1,9 @@
 
-<div class="card" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url('<%= image_path menu.photo %>')">
+<% if menu.photos? %>
+  <div class="card" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url('<%= cl_image_path menu.photos[0].path %>')">
+<% else %>
+  <div class="card" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url('http://placehold.it/350x150')">
+<% end %>
```

Comment:
<% if menu.photos? %>
        <% image = cl_image_path menu.photos(0).path %>
<% else %>
        <% image = 'http://placehold.it/350x150' %>
<% end %>

<div class="card" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url(<%= image %>)" >

The variable "image" is set before by the if/else statement, so you don't have to repeat the background-image div. Easier to update the banner if we want to change the gradient/photo etc.